tests/ops/api/v1/endpoints/test_connection_config_endpoints.py

```diff
@@ -344,6 +350,16 @@ def test_patch_connections_bulk_update(
         assert email_resource.access.value == "write"
         assert "secrets" not in email_connection
 
+        manual_webhook_connection = response_body["succeeded"][9]
```

Comment:
Is order guaranteed here? It will always be index 9?

Reply:
It is actually! for this existing endpoint, connections to patch are passed in through a list, and we loop through that list in order and add them to another list for the response.  Order stays the same unless one of the connections fail.